how-to-guides/sequential-pipelines/scripts/model_validation.py

```diff
@@ -0,0 +1,130 @@
+import neptune.new as neptune
+import neptune.new.integrations.sklearn as npt_utils
+from neptune.new.exceptions import NeptuneException
```

Comment:
```suggestion
from neptune.new.exceptions import ModelVersionNotFound
```

Reply:
I see your point, but this exception is not documented.

Only NeptuneException which is the parent and works for the majority of cases.

https://docs.neptune.ai/help/setting_up_error_handling_function/